src/index.html

```diff
@@ -1,12 +1,297 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0"
+    >
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
+    <link rel="stylesheet" href="styles/header.css">
+    <link rel="stylesheet" href="styles/product-card.css">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="/" class="logo">
+        <img
+          src="images/logo.png"
+          alt="Логотип"
+        >
+      </a>
+      <nav class="header_menu">
+        <a href="/" class="item is-active">apple</a>
+        <a href="/" class="item">samsung</a>
+        <a href="/" class="item">смартфоны и телефоны</a>
+        <a
+          href="/"
+          class="item"
+          data-qa="nav-hover"
+        >
+          ноутбуки и компьютеры
+        </a>
+        <a href="/" class="item">гаджеты</a>
+        <a href="/" class="item">планшеты</a>
+        <a href="/" class="item">фото</a>
+        <a href="/" class="item">видео</a>
+      </nav>
+    </header>
+    <main>
+      <section class="container">
+        <div class="product-card" data-qa="card">
+          <div class="product-card__image">
+          </div>
+          <p class="product-card__title">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</p>
+          <p class="product-card__code">Код товара: 195434</p>
+          <div class="product-card__rating">
+            <div class="product-card__stars product-card__stars--4">
+              <div class="product-card__star">
+              </div>
+              <div class="product-card__star">
+              </div>
+              <div class="product-card__star">
+              </div>
+              <div class="product-card__star">
+              </div>
+              <div class="product-card__star">
+              </div>
+            </div>
+            <p class="product-card__review">Отзывов: 5</p>
+          </div>
+          <div class="product-card__price-row">
+            <p class="product-card__price-tag">Цена:</p>
+            <p class="product-card__price">69 999 грн</p>
```

Comment:
use spans for this elements, not paragraphs

Reply:
+